contrib/corrosion-cmake/config.toml.in

```diff
@@ -1,4 +1,5 @@
 [env]
+# This is unused right now
 CC = "@CMAKE_C_COMPILER@"
 CXX = "@CMAKE_CXX_COMPILER@"
```

Comment:
Hm, are you sure that it is not used by i.e. cxx bindings for skim?

Reply:
You are right. It seems the problem is that the compiler is not being picked up, but cflags are